resources/wlcg-scope-tag-migration/WLCGScopeTagList.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/*
+ * A simple script to home some variables useful for the WLCG Scope tag
+ * migration.
+ */
+
+$wlcgScopesList = ["alice", "atlas", "belle", "cms", "dune", "lhcb"];
+$tierScopesList = ["tier0", "tier1", "tier2", "tier3"];
+
+// Retrieve all scope objects from the DB and store as an dict, with the
+// name as the key, for later reference.
+$scopeDql = "SELECT s FROM Scope s";
```

Comment:
Should the comment be any more specific that this is a DQL query?

Reply:
addressed in https://github.com/GOCDB/gocdb/pull/548/commits/649dc143212cb46b12b2d8e3cd18c6d596cda4b9.